app/routers/create_event.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+from datetime import datetime
+import os, sys; sys.path.append(os.path.dirname(os.path.realpath(__file__)))
+
+
+
+from fastapi import FastAPI, Request, Form
+from fastapi.responses import HTMLResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+# from pydantic import BaseModel
+
+
+
+
+app = FastAPI()
+
+# app.mount("/static", StaticFiles(directory="static"), name="static")
+
+
+templates = Jinja2Templates(directory="../templates")
+
+
+@app.get("/profile/EditEvent", response_class=HTMLResponse)
+async def insert_info(request: Request):
+    return templates.TemplateResponse("editevent.html",{"request": request})
+
+
+@app.post("/profile#id={user_id}/EditEvent")
+def create_item(user_id: int, event_title: str = Form(...), location: Optional[str] = Form(None), from_date: Optional[datetime] = Form(...),
+                to_date: Optional[datetime] = Form(...), link_vc: str = Form(None), content: str = Form(None), repeated_event: str = Form(None)):
```

Comment:
Please add return type. Also, keep in mind that having function with more than 3 arguments is usually a code smell. Can you please address it?

Reply:
Sorry, I don't understand..
if you mean that I receive json\kwargs that not possible because the setting on receive data from 'form' in fastapi see here - https://fastapi.tiangolo.com/tutorial/request-forms/